OpenRA.Game/Graphics/Viewport.cs

```diff
@@ -62,7 +62,7 @@ public float Zoom
 
 			set
 			{
-				zoom = value;
+				zoom = value.Clamp(0.25f, 2f);
```

Comment:
This is a left-over from my mouse-wheel scrolling which did not work out. I will remove it.